source/macos/AdaptiveCards/AdaptiveCards-bridge/AutoGenerated/Models/ACSTextInput.mm

```diff
@@ -140,8 +140,60 @@ - (void)setRegex:(NSString * _Nonnull)value
     mCppObj->SetRegex(valueCpp);
     
 }
+@end
 
+//// MARK: Added for maxlength
+//
+//
+//@implementation CustomTextFieldFormatter : NSFormatter
+//
+//- (id)init {
+//
+//   if(self = [super init]){
+//
+//      maxLength = INT_MAX;
+//   }
+//
+//  return self;
+//}
+//
+//- (void)setMaximumLength:(int)len {
+//  maxLength = len;
+//}
+//
+//- (int)maximumLength {
+//  return maxLength;
+//}
```

Comment:
All these comments can be removed.
Ideally we shouldn't change anything in ACS* files until or unless really required.